src/main/kotlin/it/unibo/protelis/web/execution/simulated/SimulatedProtelisEngine.kt

```diff
@@ -0,0 +1,149 @@
+package it.unibo.protelis.web.execution.simulated
+
+import it.unibo.alchemist.core.implementations.Engine
+import it.unibo.alchemist.core.interfaces.Simulation
+import it.unibo.alchemist.core.interfaces.Status
+import it.unibo.alchemist.loader.YamlLoader
+import it.unibo.alchemist.model.implementations.positions.Euclidean2DPosition
+import it.unibo.alchemist.model.implementations.times.DoubleTime
+import it.unibo.alchemist.model.interfaces.Position
+import it.unibo.protelis.web.execution.CoroutineProtelisEngine
+import it.unibo.protelis.web.execution.ProtelisObserver
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicReference
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.async
+import kotlinx.coroutines.future.asCompletableFuture
+import kotlinx.coroutines.launch
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class SimulatedProtelisEngine() : CoroutineProtelisEngine() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [SimulatedProtelisEngine] at /src/src/main/kotlin/it/unibo/protelis/web/execution/simulated/SimulatedProtelisEngine.kt:22:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)